src/reducers/transactionReducer.ts

```diff
@@ -64,6 +63,14 @@ export default function transactionReducer(state: TransactionState, action: Tran
 
     case TransactionActionTypes.SET_TRANSFER_AMOUNT: {
       const { transferAmount, chainDecimals } = action.payload;
+      if (!transferAmount) {
+        return {
+          ...state,
+          transferAmount,
+          transferAmountError: null,
+          transactionReadyToExecute: false
+        };
+      }
```

Comment:
How is this related?

Reply:
https://github.com/paritytech/parity-bridges-ui/pull/248#discussion_r679939981